scripts/regressor_finder.py

```diff
@@ -500,7 +503,7 @@ def results():
                     exc = future.exception()
                     if exc is not None:
                         logger.info(
-                            f"Exception {exc} while analyzing {futures[future]}"
+                            "Exception %d while analyzing %s", exc, futures[future]
```

Comment:
The value of `exc` is an Exception, not an integer.

Reply:
Yeah fixed that!!!